specification/dimensions/chargedescription.md

```diff
@@ -0,0 +1,30 @@
+# Charge Description
+
+A Charge Description provides a high-level context of a line item without requiring additional discovery.  This column is typically a brief, human-readable summary of a select group of corresponding details across a cost and usage dataset or provides information not otherwise available.
+
+The Charge Description column MUST be present within billing data, MUST be of type String, and SHOULD NOT be null.
```

Comment:
```suggestion
The Charge Description column MUST be present within billing data, MUST be of type String, and SHOULD NOT be null.  Providers SHOULD specify the length of this column in their publicly available FOCUS documentation. 
```

Reply:
Whether we specify this per column or generally in a separate section of the spec is up for debate.